src/game/UICmdList.h

```diff
@@ -0,0 +1,60 @@
+// UICmdListDlg.h: interface for the CUICmdListDlg class.
+//
+//////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "N3Base/N3UIBase.h"
+#include "N3Base/N3UIButton.h"
+#include "N3Base/N3UIList.h"
+#include "N3Base/N3UIEdit.h"
+
+//////////////////////////////////////////////////////////////////////
+
+class CUICmdListDlg : public CN3UIBase {
```

Comment:
As with the other UI class, officially the class name is simply named `CUICmdList`.

I don't know if this project intends to keep official class names for things like this, but if so, this should be renamed `CUICmdList`.